lib/utils/module.js

```diff
@@ -0,0 +1,39 @@
+var fs = require('fs');
+var debug = require('debug')('axm:profiling');
+var path = require('path');
+
+var moduleUtils = {
+  detectModule: function (moduleName, cb) {
+    require.main = require.main || {paths: ['./node_modules', '/node_modules']};
+
+    var requirePaths = require.main.paths.slice();
```

Comment:
Is `require.main.paths` still available with MJS module ?

Reply:
Still available on all Node version. Not tested with .mjs file.